source/renderer/zettlr-editor.js

```diff
@@ -27,6 +27,7 @@ const { trans } = require('../common/lang/i18n.js')
 const generateKeymap = require('./assets/codemirror/generate-keymap.js')
 const EditorSearch = require('./util/editor-search')
 const openMarkdownLink = require('./util/open-markdown-link')
+const ipc = require('electron').ipcRenderer
```

Comment:
I just saw this when looking over this again — please don't do this as of now! Currently all IPC calls are centrally handled in the IPC wrapper classes. It's not good design, I'm that fare, but I'd like to refactor and decentralise the IPCs in one block, and not piece by piece. See also my other comment on the theme switching